lemur/roles/views.py

```diff
@@ -257,6 +257,7 @@ def get(self, role_id):
         )
 
     @validate_schema(role_input_schema, role_output_schema)
+    @operator_permission.require(http_exception=403)
```

Comment:
this is gated by admin already https://github.com/Netflix/lemur/blob/1c096f1028a237a6f58d181e5aaabb0f8f1e0953/lemur/auth/permissions.py#L58